config/initializers/simple_form.rb

```diff
@@ -0,0 +1,165 @@
+# Use this setup block to configure all options available in SimpleForm.
```

Comment:
These files were added to clear the simple_form warning messages.  Running the generator for simple form doesn't impact any of our stuff (online it said it uses the defaults anyway if you don't run the generator).  Is that okay?